py/README_TEMPLATE.rst

```diff
@@ -0,0 +1,171 @@
+
+
+What is PySparkling?
+--------------------
+
+PySparkling is an integration of Python with Sparkling Water. It allows user to start H2O services on a Spark cluster from Python API.
+  
+In the PySparkling driver program, the Spark Context, which uses Py4J to start the driver JVM and the Java Spark Context, is used to create the H2O Context (hc).  That in turn starts an H2O cloud (cluster) in the Spark ecosystem. Once the H2O cluster is up, the H2O Python package is used to interact with it and run H2O algorithms. All pure H2O calls are executed via H2O's REST API interface. Users can easily integrate their regular PySpark workflow with H2O algorithms using PySparkling.
+  
+PySparkling programs can be launched as an application or in an interactive shell or notebook environment. 
+  
+
+PySparkling and Spark Version
+-----------------------------
+
+There are multiple PySparkling packages, each is intended to be used with different Spark version.
+
+SUBST_VERSIONS_TABLE
+
+So for example, to install PySparkling for Spark SUBST_SPARK_MAJOR_VERSION, the command would look like:
+
+.. code-block:: bash
+
+    pip install h2o_pysparkling_SUBST_SPARK_MAJOR_VERSION
+
+Dependencies
+------------
+
+Prerequisites:
+    
+  - Python 2.7 or 3+
+  - Numpy 1.9.2
+
+For Windows users, please grab a .whl from http://www.lfd.uci.edu/~gohlke/pythonlibs/#numpy.
+
+In order to use PySparkling, it requires the following runtime python dependencies to be available on the system, all of which are available on PyPI:
+
+.. code-block:: bash
+
+  $ pip install requests
+  $ pip install tabulate
+  $ pip install future
+  $ pip install colorama>=0.3.8
+
+The required packages are installed automatically in case when PySparkling is installed from PyPI.
+
+
+The Sparkling Water Python Module
+---------------------------------
+
+Prepare the environment
+~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Either clone and build Sparkling Water project
+
+  .. code-block:: bash
+
+    git clone http://github.com/h2oai/sparkling-water
+    cd sparkling-water
+    ./gradlew build -x check
+
+
+ or download and unpack sparkling water release from  `here <https://www.h2o.ai/download/>`_.
+
+2. Configure the location of Spark distribution and cluster:
+
+  .. code-block:: bash
+
+    export SPARK_HOME="/path/to/spark/installation"
+    export MASTER='local[*]'
```

Comment:
Isn't this configuration just for developers running Spark on laptops?

Reply:
Yup, this doc should be for a quick start. We can mention also yarn etc. I would also do that in another JIRA as it's not target of this one, what do you think? :)